docs/ansible-release-announcement/minor-release/announcement-email.md

```diff
@@ -0,0 +1,84 @@
+Release announcement : Ansible Community Package X.0.0
+
+Hello everyone,
+
+We're happy to announce the release of the Ansible X.0.0 package!
+
+Ansible X.0.0 requires latest version of ansible-core a.b and includes a curated set of Ansible collections that provides a vast number of modules and plugins. This is a major release of Ansible.
+
+ansible-core is a required dependency, not contained within the ansible packages. Pip install builds the
+dependency, but it can be built and installed quite independently of the "ansible" distribution.
+
+How to get it
+-------------
+
+This release is available on PyPI and can be installed with pip:
+
+`$ python3 -m pip install ansible==X.0.0 --user`
+
+The sources for this release can be found here:
+
+Release tarball:
+
+
+SHA256:
+
+
+
+Wheel package:
+
+
+
+SHA256:
+
+
+
+Some important details
+----------------------
+
+Python wheels are now available for both Ansible X and ansible-core a.b resulting in significantly improved installation performance.
+
+In addition, Ansible x no longer installs some unnecessary files from the included Ansible collections such as tests or hidden files and directories. This further improves installation performance and reduces the size on disk. These files are still available in the source tarball if needed.
+
+Collections which have opted-in to being a part of the Ansible 8 unified changelog will have an entry on this page:
+
+For collections which have not opted-in to the unified changelog, you may find more information on https://docs.ansible.com/ansible/latest/collections or the collection source repository. For example, the community.crypto collection is available at https://docs.ansible.com/ansible/latest/collections/community/crypto/index.html and you can find a link to the source repository under the "Repository(Sources)" button.
+
+The changelog for ansible-core a.b installed by this release of Ansible X can be found here: <changelog of ansible-core>.
+
+
+What's the schedule for new Ansible releases after X.0.0?
+---------------------------------------------------------
```

Comment:
RST syntax in Markdown?

Reply:
This is actually MarkDown (CommonMark): https://spec.commonmark.org/0.31.2/#setext-headings